### PR TITLE
Sync .auto branch with the latest branch in internal repository

### PR DESCRIPTION
**Pull request by wep21**
## Description

Sync .auto branch with latest branch in internal repository 

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@wep21 Could you rebase on the [main branch](https://github.com/tier4/AutowareArchitectureProposal.iv/commits/main) or merge main? :pray:

**Comment by tkimura4** [comment]
@1222-takeshi 
git merge origin/use-autoware-auto-msgs
としてから
こちら、ビルド確認/psim確認お願いできますか?

**Comment by 1222-takeshi** [comment]
ビルド、PSim共に動作確認できました。
psimもengageまで問題なく動いてそうです。

**Comment by wep21** [comment]
@yukke42 centerpointちょっと変えたので問題ないか確認してもらえると🙏

File: perception/object_recognition/detection/lidar_centerpoint/src/node.cpp [hunk 1]
```diff
@@ -84,46 +87,33 @@ void LidarCenterPointNode::pointCloudCallback(
...
     classification.probability = 1.0f;
-    switch (class_id) {
-      case 0:
-        classification.label = autoware_auto_perception_msgs::msg::ObjectClassification::CAR;
+    classification.label = getSemanticType(class_names_[class_id]);
 
+    using Label = autoware_auto_perception_msgs::msg::ObjectClassification;
+    if (classification.label == Label::CAR) {
+      obj.kinematics.orientation_availability =
+        autoware_auto_perception_msgs::msg::DetectedObjectKinematics::SIGN_UNKNOWN;
```
Comment (yukke42) [inline perception/object_recognition/detection/lidar_centerpoint/src/node.cpp:107]: @wep21 
TRUCKとBUSの場合もSIGN_UNKNOWNにするよう追加していただけますか :pray: 
それ以外はOKです！

Comment (wep21) [reply perception/object_recognition/detection/lidar_centerpoint/src/node.cpp]: @yukke42 fixed at https://github.com/tier4/AutowareArchitectureProposal.iv/pull/691/commits/109421c875cec5716cfa67c5c12bd18aead14ef0

**Review by tkimura4 — APPROVED** [review]
LGTM

**Review by yukke42 — APPROVED** [review]

**Review by 1222-takeshi — APPROVED** [review]
ビルド、動作確認できました

**Comment by tkimura4** [comment]
image
これってsquashマージで良いんですかね
@wep21  @kenji-miyake

**Comment by kenji-miyake** [comment]
@tkimura4 1個1個revertする可能性があるならRebase and mergeが良いですが、そうじゃないならSquashでも大丈夫です。Rebase and mergeをする場合は有効化します。

でもRebaseにするなら、事前にここはSquashしておきたい感じがありますね。
image

**Pull request merged**